cmd/operator/main.go

```diff
@@ -32,13 +33,15 @@ func main() {
 		namespace       string
 		metricsAddr     string
 		healthProbeAddr string
+		images          k8sflag.MapStringString
 
 		setupLog = ctrl.Log.WithName("setup")
 	)
 
 	flag.StringVar(&namespace, "namespace", "default", "The namespace in which the operator runs")
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&healthProbeAddr, "health-probe-bind-address", ":8081", "The address the health probe endpoint binds to.")
+	flag.Var(&images, "images", "Full images refs to use for containers managed by the operator. E.g thanos=quay.io/thanos/thanos:v0.33.0")
```

Comment:
We'd need to enumerate which keys are supported (e.g. "prometheus", "alertmanager" and "thanos") and the operator should bail out if a key isn't in the list.